security-proxy/headers-mapping.properties

```diff
@@ -1,15 +1,91 @@
-# Keys are additional headers sent by the security-proxy,
-# mapped onto user's LDAP attributes.
+# Keys are additional headers sent by the security-proxy, mapped onto user's LDAP attributes:
+# Additionally, LDAP objects related to the authenticated user are accessible through
+# prefixed attribute names `manager.`, `org.`, and `org.seeAlso.`
+# 
+# Header names with no prefix (default headers) apply to all proxified services (e.g. sec-email=mail).
+# Header names prefixed with a service name as defined in targets-mapping.properties
+# complement the default headers when hitting a specific proxified servicei. for example:
+# analytics.sec-org-address=org.seeAlso.postalAddress
+# will also add the `sec-org-address` with the user organization's postal address as value
+# to requests hitting the analytics service.
+# 
+# Base64 header value encoding:
+# ============================
+# Due to HTTP headers being ISO-8859-1 encoded by spec, header values that can potentially have
+# characters outside that encoding can be configured to be base64-encoded. If so, the target service
+# must know how to decode them. For example:
+# sec-orgname=base64:org.o
+# will override the default sec-orgname header with a base64 encoded value.
 #
-# The following headers are already provided by the proxy,
-# regardless of this file's content:
-#  * sec-username
-#  * sec-roles
-#  * sec-org
-#  * sec-orgname
-#  * sec-proxy
```

Comment:
`sec-proxy` header disappeared ?

Reply:
it is still added here: https://github.com/georchestra/georchestra/blob/a2b75a7c662f4da09b50b7d1375bb504479a2ee3/security-proxy/src/main/java/org/georchestra/security/HeadersManagementStrategy.java#L96 but does not require a mapping to a LDAP field.